fe/spark-dpp/src/main/java/org/apache/doris/load/loadv2/dpp/SparkRDDAggregator.java

```diff
@@ -240,11 +242,19 @@ Object update(Object dst, Object src) {
     }
 }
 
+class ReplaceIfNullAggregator extends SparkRDDAggregator<Object> {
+
+    @Override
+    Object update(Object dst, Object src) {
+        return dst == null ? src : dst;
+    }
+}
+
 class ReplaceIfNotNullAggregator extends SparkRDDAggregator<Object> {
 
     @Override
     Object update(Object dst, Object src) {
-        return src == null ? dst : src;
+        return dst != null ? src : dst;
```

Comment:
hi @wangbo 
could you please help double review logic of this change please
Regards